src/words.jl

```diff
@@ -92,7 +94,7 @@ Base.@propagate_inbounds function Base.setindex!(w::Union{Word, SubWord}, v::Int
 end
 
 """
-    longestcommonprefix(u::Word, v::Word)
+    longestcommonprefix(u::Union{Word, SubWord}, v::Union{Word, SubWord})
```

Comment:
```suggestion
    longestcommonprefix(u::AbstractWord, v::AbstractWord)
```
whenever we don't use `ptrs` field the method should be applicable to `AbstractWord`s